### PR TITLE
Sga 53 walking activity indicators implementation

### PR DESCRIPTION
**Pull request by ugnspa**
- Changed UI in activity screen: circular progress bar for steps and two counters for distance and calories
- Implemented Health connect service which read calories and distance
- Calories and distance are now displayed from Health connect and not calculated manually
screen-15 10 49 14 03 2024

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -34,6 +36,8 @@ lifecycleRuntimeCompose = "2.7.0"
 
 [libraries]
 accompanist-permissions = { module = "com.google.accompanist:accompanist-permissions", version.ref = "accompanist" }
+androidx-connect-client = { module = "androidx.health.connect:connect-client", version.ref = "connectClient" }
```
Comment (mantasjasikenas) [inline gradle/libs.versions.toml:39]: Dependency alias 'androidx-connect-client' is not used in build scripts. Remove this.

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -34,6 +36,8 @@ lifecycleRuntimeCompose = "2.7.0"
...
+androidx-connect-client-v110alpha07 = { module = "androidx.health.connect:connect-client", version.ref = "androidx-connectClient" }
```
Comment (mantasjasikenas) [inline gradle/libs.versions.toml:40]: Rename this to 'androidx-connect-client'

File: app/src/main/java/com/github/k409/fitflow/features/stepcounter/CaloriesAndDistanceUtil.kt [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package com.github.k409.fitflow.features.stepcounter
+
+import android.util.Log
+import com.github.k409.fitflow.di.services.HealthConnectService
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneId
+import javax.inject.Inject
+
+class CaloriesAndDistanceUtil @Inject constructor(
```
Comment (mantasjasikenas) [inline app/src/main/java/com/github/k409/fitflow/features/stepcounter/CaloriesAndDistanceUtil.kt:10]: I would suggest renaming this class to enhance its clarity and expressiveness. A possible name could be `HealthStatsManager` or something similar.

**Review by mantasjasikenas — APPROVED** [review]

**Comment by ugnspa** [comment]
changes

**Pull request closed without merging**